src/assets/config.json

```diff
@@ -40,6 +40,8 @@
 
   "satisfaction": 100,
   "expectedDelivery": 7,
+  "weightDeliverySwiftness": 0.5,
```

Comment:
Perhaps one variable, say `lambda`, and then have weights `lambda` and `1 - lambda` in the function?